src/common/screens/reducer.js

```diff
@@ -4,7 +4,7 @@ import {Record} from 'immutable';
 // We can use simple initialState if no data from server need to be revived.
 const InitialState = Record({
   currentScreen: 0,
-  lastScreen: 5
+  lastScreen: 8
```

Comment:
Can't we calculate it on the fly? In sense: the last screen is the one that doesn't have any transitions?